etc/omero.properties

```diff
@@ -819,15 +819,19 @@ omero.client.scripts_to_ignore=/omero/figure_scripts/Movie_Figure.py,\
 /omero/figure_scripts/ROI_Split_Figure.py,/omero/export_scripts/Make_Movie.py,\
 /omero/setup_scripts/FLIM_initialise.py,/omero/import_scripts/Populate_ROI.py
 
+# Flag to show/hide "Orphaned images" container. Only accept "true" or "false"
+omero.client.ui.tree.orphans.enabled=true
 # Name of the "Orphaned images" container located in client tree data manager.
-omero.client.ui.tree.orphans.name=Orphaned images
-
+omero.client.ui.tree.orphans.name=Orphaned Images
 # Description of the "Orphaned images" container.
 omero.client.ui.tree.orphans.description=This is a virtual container with orphaned images. These images are not linked anywhere. Just drag them to the selected container.
 
+# helper combining all orphans properties. Do not set manually that property.
+omero.client.ui.tree.orphans={"enabled": "${omero.client.ui.tree.orphans.enabled}", "name": "${omero.client.ui.tree.orphans.name}", "description": "${omero.client.ui.tree.orphans.description}"}
```

Comment:
@sbesson thanks for looking. We have discussed this morning with @joshmoore that is not necessary to add wrapping property. It should use ``getClientConfigValues``. I am about to open yet another PR. Hopefully this will be the final solution.